db/migrate/20190321195704_create_procedure_instances.rb

```diff
@@ -0,0 +1,12 @@
+class CreateProcedureInstances < ActiveRecord::Migration[5.2]
+  def change
+    create_table :procedure_instances do |t|
+      t.integer :procedure_id
+      t.float :price
+      t.string :status
+      t.date :date
+      t.integer :facility_id
```

Comment:
use a `references` here too and place the foreign key columns at the top of the table.

Reply:
I don't think that the `references` need to be at the top of the table; [here's a Rails Guides example](https://guides.rubyonrails.org/association_basics.html#creating-foreign-keys-for-belongs-to-associations) where `t.references` is at the bottom.